### ISSUE TITLE
Filza Electracuted

### ISSUE THREAD

**Issue by MrRobotjs** [kdog0598/kdog0598.github.com#1]
Did you remove it from your repo? If so why?

**Comment by kdog0598** [comment]
yeah because it was only needed in the early days of the electra jailbreak when there wasn't a readily available filza.
